<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addpartsToDialog</class>
 <widget class="QDialog" name="addpartsToDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Наименование</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_part_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Количиство, шт</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_qty">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_save_parts">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>СОХРАНИТЬ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_cancel_parts">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ОТМЕНА</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_delivery_date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>дата поставки</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="de_delivery_date">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumDate">
    <date>
     <year>2100</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>1970</year>
     <month>9</month>
     <day>14</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_cancel_parts</sender>
   <signal>released()</signal>
   <receiver>addpartsToDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_save_parts</sender>
   <signal>released()</signal>
   <receiver>addpartsToDialog</receiver>
   <slot>onSavePartClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSavePartClicked()</slot>
 </slots>
</ui>
